definitions/tools/picard_merge_vcfs.cwl

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Picard MergeVcfs"
+baseCommand: ["/usr/bin/java", "-jar", "/opt/picard/picard.jar", "MergeVcfs"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 40000
```

Comment:
If we're reserving this much memory for this job, should we be setting `-Xmx36g` or the like for the java options, too?

Reply:
Is that much RAM really required, or was that a typo and we really only need 4GB. I can't imagine needing 40GB.

Reply:
The memory requirement was set when we were merging the chromosomal.g.vcf's that had variant calls at every position in the genome. With an exome interval list, the memory requirement should be a lot less. However, I have not tested what the low end should be. Is there a way to estimate this value?

Reply:
Yes, it turns out that much memory really is needed by this tool sadly.  Current tests with real data are failing after only processing chr1 and part of chr2

Reply:
We've run across another failure due to running out of memory; bumping it back up with #956